sdk/storage/Microsoft.Azure.WebJobs.Extensions.Storage.Common/src/Shared/Queues/QueueListener.cs

```diff
@@ -113,7 +126,7 @@ public QueueListener(QueueClient queue,
 
             // if the function runs longer than this, the invisibility will be updated
             // on a timer periodically for the duration of the function execution
-            _visibilityTimeout = TimeSpan.FromMinutes(10);
+            _visibilityTimeout = _queueOptions.VisibilityTimeout;
```

Comment:
I don't think you need to make these changes in this PR. I agree this is somewhat confusing but is discussed more in detail in https://github.com/Azure/azure-webjobs-sdk/issues/1040. Summary is, that the configured QueuesOptions.VisibilityTimeout IS used when messages fail processing (see [code](https://github.com/Azure/azure-webjobs-sdk/blob/db0bfcdff6e0855ddadf08f74fd5007af9283da5/src/Microsoft.Azure.WebJobs.Extensions.Storage/Queues/QueueProcessor.cs#L125)). This hardcoded value is just the initial timeout used when dequeuing messages. If we want to make any changes to this, I think we should discuss and do that separately, because it's not really related to the cancellation token changes right?

Reply:
That's fair, it is indeed a separate scenario, I'll remove from this PR :)